x-pack/plugins/synthetics/public/apps/synthetics/components/monitors_page/management/labels.ts

```diff
@@ -24,22 +24,22 @@ export const LEARN_MORE_LABEL = i18n.translate(
 export const CALLOUT_MANAGEMENT_DISABLED = i18n.translate(
   'xpack.synthetics.monitorManagement.callout.disabled',
   {
-    defaultMessage: 'Monitor Management is disabled',
+    defaultMessage: 'Monitor Management is currently disabled',
   }
 );
 
 export const CALLOUT_MANAGEMENT_CONTACT_ADMIN = i18n.translate(
   'xpack.synthetics.monitorManagement.callout.disabled.adminContact',
   {
-    defaultMessage: 'Please contact your administrator to enable Monitor Management.',
+    defaultMessage: 'Monitor Management will be enabled when an admin visits the Synthetics app.',
```

Comment:
@florent-leborgne I have updated the copy from `Please contact your administrator to enable Monitor Management` to the above.

We have recently merged a feature where monitor management is enabled automatically in the background when an admin visits the app. Is this clear in this copy? It'll be documented as well, of course.